atecc608a/main.c

```diff
@@ -103,7 +103,7 @@ enum {
     key_bits = 256,
     hash_alg = PSA_ALG_SHA_256,
     alg = PSA_ALG_ECDSA(hash_alg),
-    sig_size = PSA_ASYMMETRIC_SIGN_OUTPUT_SIZE(key_type, key_bits, alg),
```

Comment:
In "Set signature size directly", could you copy and paste the error or warning message into the commit message? It'll help to make it more clear why this failed in IAR.

Reply:
Done